ktx

```diff
@@ -15,7 +15,10 @@
 # limitations under the License.
 
 main() {
-    # TODO: ensure the file actually exists.
-    echo "export KUBECONFIG=\"${HOME}/.kube/${1}\""
+    if [ -e ${HOME}/.kube/${1} ];then
+        echo "export KUBECONFIG=\"${HOME}/.kube/${1}\""
+    else
+        echo "echo \"The following file does not exist: ${HOME}/.kube/${1}. Exiting...\""
```

Comment:
Let's add a comment to the top of the function explaining why we have to echo "echo". This could be confusing in future.